monitoring/Troubleshooting.md

```diff
@@ -0,0 +1,34 @@
+# Troubleshooting
+
+## Issue: Manually deleting the monitoring namespace does not delete all components 
+
+### Description
+The monitoring components should be removed by using the `remove_monitoring_cluster.sh` and 
+`remove_monitoring_viya.sh` scripts. See [Remove Monitoring Components](README.md#mremove) for more information. If you attempt to remove the monitoring components by only deleting the namespace into which the components are deployed, components in other locations are not removed and redeployment of monitoring fails. 
+
+### Solution
+Run these commands to delete all of the monitoring components that are in locations 
+other than the monitoring namespace.
+
+```bash
```

Comment:
We need to better handle line breaks here. Each command separated by spaces needs to either be on a single long line or have the `\` line-continuation character at the end of each line.